apps/client/src/sttr_client/models/checker.js

```diff
@@ -104,6 +104,42 @@ class Checker {
     return this.stack[this.stack.length - 1];
   }
 
+  /**
+   * find all permits that need contact,
+   * for evert permit see if the decisiveQuestion for contact
+   * equals the current (recently answered) question
```

Comment:
1. Can you update the comment?
```
 * Find all permits that have a "contact" conclusion,
 * for every permit see if the decisiveQuestion for contact
 * equals the current (recently answered) question
```

2. You are referring to `decisiveQuestion` which is a variable in our client, does that make sense?